test-app/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.8"
+services:
+  test-app:
+    image: lblod/embeddable-say-editor-test:v3.4.0
```

Comment:
Another option here is to not push the `Dockerfile` to the registry at all, and link to the local `Dockerfile` here using the `build` specification (https://docs.docker.com/compose/compose-file/build/).
I'm not sure how important it is to push seperate versioned versions of the `test-app` to docker-hub.

Reply:
Hmm, true. Was thinking to avoid having to run the build on the dev/QA servers, but I don't actually know how resource constrained those machines are, so maybe we could just do that.

Reply:
The advantage of this is also that we do not have to publish a release in order to deploy on the DEV environment (we can just use latest master). It would also reduce the pressure on the CI.